Rename to HTTP Datagrams (#64)
ietf-wg-masque/draft-ietf-masque-h3-datagram: draft-ietf-masque-h3-datagram.md

# Multiplexing

When running over HTTP/3, multiple exchanges of datagrams need the ability to
coexist on a given QUIC connection. To allow this, HTTP datagrams contain two
layers of multiplexing. First, the QUIC DATAGRAM frame payload starts with an
encoded stream identifier that associates the datagram with a given QUIC
stream. Second, datagrams optionally carry a context identifier (see
{{datagram-contexts}}) that allows multiplexing multiple datagram contexts
related to a given HTTP request. Conceptually, the first layer of multiplexing
is per-hop, while the second is end-to-end.

When running over HTTP/2, the first level of demultiplexing is provided by the

> HTTP/2 framing layer. When running over HTTP/1, only one request can exist on
> an HTTP connection at a given time, therefore the first layer of demultiplexing
> is not needed.

Reviewer: alternatively you could say 

```suggestion
HTTP/2 framing layer. When running over HTTP/1, requests are strictly serialized
in the connection, therefore the first layer of demultiplexing is not needed.
```
